decidim-api/spec/tasks/decidim_api_generate_docs_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "rake decidim_api:generate_docs", type: :task do
+  it "creates the static docs files" do
+    static = Rails.root.join("app/views/static/api/docs")
```

Comment:
Should we add something in the CHANGELOG? It could be weird that for a Decidim upgrade, apart from the typical file changes that we usually do (`Gemfile, Gemfile.lock, db/migrate/*, db/schema.rb`) that we also add these files without saying anything to implementers.

Reply:
![image](https://user-images.githubusercontent.com/105683/149167477-1bf8f823-003c-4321-b96d-1f295413ca20.png)